stingray/data/sample

```diff
@@ -0,0 +1,16 @@
+ccopy_reg
+_reconstructor
+p0
+(c__main__
+Usman
+p1
+c__builtin__
+object
+p2
+Ntp3
+Rp4
+(dp5
+S'x'
+p6
+I10
+sb.
```

Comment:
No, this is not required at all. I just included it for demonstration purposes. This can be safely discarded.